bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bootstrap-select-to-button",
-  "main": "src/main.js",
+  "main": "src/bootstrapSelectToButton.js",
```

Comment:
We should prefix this with jquery, eg. `jquery.bootstrapSelectToButton.js`.  Same with the .min version.